src/components/CreatePost/CreatePost.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+
+import { kwitterApiClient } from '../../api/client';
+import { KweetsList } from '../../components/KweetsList';
+import { Kweet as KweetModel } from '../../model/kweet';
+import { ListKweetsResponse } from '../../model/kwitterApiModels';
+import {
+  KwitterButton,
+  KwitterPost,
+  KwitterTextArea,
+  ProfilePhoto
+} from './styled';
+
+export const CreatePost = (): React.ReactElement => {
+  const [post, setPost] = useState<string>('');
+  const [kweetList, setKweetList] = useState<KweetModel[]>([]);
+
+  const fetchData = async () => {
+    const fetchKweets = await kwitterApiClient.get<ListKweetsResponse>(
+      '/kweets'
+    );
+    setKweetList(fetchKweets.data);
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const kweet = {
```

Comment:
Change request: You could move this declaration into the submit handler function so you don't unnecessarily declare it for every render where you won't be using it.

Reply:
yeah good point!